wcivf/apps/elections/management/commands/import_posts.py

```diff
@@ -63,3 +64,24 @@ def populate_any_non_by_elections_field(self):
             )
             election.any_non_by_elections = any_non_by_elections
             election.save()
+
+    def extract_territory(self, ee_data):
+        if ee_data and ee_data['organisation']:
+            return ee_data['organisation'].get('territory_code', '???')
+
+    def import_territories(self):
+        ee = EEHelper()
+
+        post_elections_without_territory = PostElection.objects.filter(post__territory='')
+
+        for post_election in post_elections_without_territory:
+
+            ee_data = ee.get_data(post_election.ballot_paper_id)
+
+            territory = self.extract_territory(ee_data)
+
+            print("%s - %s" % (post_election.ballot_paper_id, territory))
```

Comment:
Couple of points here:

1. We try (but don't always, for legacy reasons) to use `self.stdout.write` in management commands – this makes testing them easier as you can [replace the classes stdout class](https://github.com/DemocracyClub/UK-Polling-Stations/blob/03d1378103a0921b329edccf1abeeb8e611e9429/polling_stations/apps/councils/tests/test_council_importer.py#L58).

2. Having said that, I'm not sure if this is useful beyond initial debugging?

Reply:
Gotcha, happy to make that change.

I included it mainly as a feedback mechanism - other management commands list out e.g. parties, posts as they're imported, and it felt right to include a way of demonstrating that something is actually happening.

Happy to change the copy to something cleaner too like "Adding territory to some.ballot.id - NIR"

Reply:
At right, yes – ok let's keep the output, but convert to `self.stdout` 👍

Reply:
These changes have been added in, merges resolved, and migrations updated to avoid conflicts with migrations made in the meantime.